src/sass/abstract/_mixin.scss

```diff
@@ -0,0 +1,76 @@
+@mixin clearfix {
+    &::after {
+        content: '';
+        display: table;
+        clear: both;
+    }
+}
+
+@mixin absCenter {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+// MEDIA QUERY MANAGER
+/*
+  0 -- 600px:     Phone
+  600 -- 900px:     Tablet portrait
+  900 -- 1200px:     Tablet landscape
+  [1200 -- 1800]px is where our normal styles apply
+  1800px:     Big desktop
+  
+  $breakpoint argument choices
+  -phone
+  -tab-port
+  -tab-land
+  -big-desktop
+  
+  1em - 16px
+  */
```

Comment:
remove unused codes

Reply:
This a comment describing the function of the mixins which were for the media query, and how i achieved them

Reply:
good